### PR TITLE
Fix long file name issue for Azure PowerShell .NET Reference

### PR DESCRIPTION
**Pull request by huangmin-ms**
Fix issue: https://dev.azure.com/ceapex/Engineering/_workitems/edit/527815

### CODE REVIEW

**Comment by huangmin-ms** [comment]
Build failed reason:
It turns out that QuickIO.NET cannot handle path with forward slash ("/"), For example "C:\work\mdoc\output\test/ns-TheNamespace.xml".

**Review by yishengjin1413 — APPROVED** [review]

**Comment by huangmin-ms** [comment]
Retarget Mono.Ceil to net 471: https://docs.microsoft.com/en-us/nuget/reference/packages-config

File: mdoc/packages.config [hunk 1]
```diff
@@ -1,4 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="Mono.Cecil" version="0.10.0-beta5" targetFramework="net45" />
+  <package id="QuickIO.NET" version="2.6.2.0" targetFramework="net471" />
```
[the remark this answers is not in the document]
Comment (huangmin-ms) [reply mdoc/packages.config:4]: Retarget to net471 based on https://docs.microsoft.com/en-us/nuget/reference/packages-config

[the remark this answers is not in the document]
Comment (huangmin-ms) [reply mdoc/packages.config:4]: > why they target to different framework?

Retargetd to net471 based on https://docs.microsoft.com/en-us/nuget/reference/packages-config

**Review by daxianji007 — APPROVED** [review]

**Pull request merged**